<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogUnitCell</class>
 <widget class="QDialog" name="DialogUnitCell">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>275</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Number of peaks selected: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxNumberPeaks">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>411</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #BBBBFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #DDDDFF, stop: 1 #FFFFFF);
 }</string>
   </property>
   <property name="title">
    <string>Unit-cell</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>383</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxb">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>b</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxa">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>c</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxc">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxalpha">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxbeta">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxgamma">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonFindUnitCell">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="D19Proto.qrc">
      <normaloff>:/unitcellIcon.png</normaloff>:/unitcellIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>411</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFBBBB);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #DDDDFF, stop: 1 #FFFFFF);
 }</string>
   </property>
   <property name="title">
    <string>UB matrix</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>272</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUB01">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUB02">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUB10">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUB12">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUB20">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUB21">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUB22">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUB11">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxUB00">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonFindReindexHKL">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reindex selected peaks&lt;/p&gt;&lt;p&gt;with this UnitCell and orientation-matrix&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="D19Proto.qrc">
      <normaloff>:/IconToHKL.png</normaloff>:/IconToHKL.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGivePMatrix">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="D19Proto.qrc">
      <normaloff>:/unitcellTransformIcon.png</normaloff>:/unitcellTransformIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="D19Proto.qrc"/>
 </resources>
 <connections/>
</ui>
